communication/CMakeLists.txt

```diff
@@ -1,48 +1,43 @@
 project (libbftcommunication LANGUAGES CXX)
 
-set(bftcommunication_src
-  src/CommFactory.cpp
-  src/PlainUDPCommunication.cpp
-)
-
-if(BUILD_COMM_TCP_PLAIN)
-    set(bftcommunication_src ${bftcommunication_src} src/PlainTcpCommunication.cpp)
-endif()
-if(BUILD_COMM_TCP_TLS)
-    set(bftcommunication_src ${bftcommunication_src}
-        src/TlsTCPCommunication.cpp
-        src/TlsMultiplexCommunication.cpp
-        src/TlsRunner.cpp
-        src/TlsConnectionManager.cpp
-        src/AsyncTlsConnection.cpp
-    )
+if(BUILD_COMM_TCP_PLAIN AND BUILD_COMM_TCP_TLS)
```

Comment:
There should also be a check that at least one of all the communication methods is chosen.

Reply:
UDP is default is nothing is chosen

Reply:
I would still add it, as the check is already there